src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource_formatters.py

```diff
@@ -424,7 +424,7 @@ def test_no_color_mode(self):
       numberValue:  1.2
       booleanValue: true
       stringValue:  "The quick brown fox jumps over the lazy dog."
-"""
+"""  # noqa: F541
```

Comment:
If there is no place holder, we can remove the leading `f` instead. 

https://flake8.pycqa.org/en/latest/user/error-codes.html

F541 | f-string without any placeholders
-- | --